frontend/libs/studio-components/src/components/StudioPageError/StudioPageError.module.css

```
@@ -0,0 +1,9 @@
+.container {
+  max-width: 700px;
+  margin: 0 auto;
+  margin-top: 50px;
```

Her kan vi bruke snarveien `margin: [topp] [sider] [bunn]`. Og hva med å bruke en forhåndsdefinert avstandsvariabel?
```suggestion
  margin: var(--fds-spacing-12) auto 0;
```

---

fikset